Formula/emacs.rb

```diff
@@ -27,17 +27,19 @@ class Emacs < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gnutls"
+  depends_on "jansson"
 
   uses_from_macos "libxml2"
   uses_from_macos "ncurses"
 
+  patch :DATA
+
   on_linux do
     depends_on "jpeg"
   end
 
   def install
     args = %W[
-      --disable-dependency-tracking
```

Comment:
Is this needed? It seems unrelated

Reply:
That configure flag is no longer supported. While I was testing I saw that the configure script warns that it's unrecognized and so I removed it.